.github/CONTRIBUTING.md

```diff
@@ -32,9 +32,9 @@ our workflow that are not covered by a bot or status check are:
 Due to the fact that this project is entirely volunteer-run (i.e. no one is paid
 to work on Python full-time), we unfortunately can make no guarantees as to if
 or when a core developer will get around to reviewing your pull request.
-If no core developer has done a review or responded to changes made because of a
-"changes requested" review, please feel free to email [python-dev](https://mail.python.org/mailman3/lists/python-dev.python.org/) to ask if
-someone could take a look at your pull request.
+If no core developer has done a review or responded to changes made because of a "changes requested" review, please consider seeking assistance through the [python-core-dev](https://discuss.python.org/c/core-dev/23).
```

Comment:
```suggestion
If no core developer has done a review or responded to changes made because of a
"changes requested" review, please consider seeking assistance through the
[Core Development Discourse category](https://discuss.python.org/c/core-dev/23).
```

Reply:
Okay